lib/ngx/re.lua

```diff
@@ -0,0 +1,74 @@
+-- Copyright (C) Thibault Charbonnier (thibaultcha)
```

Comment:
Because you are granting copyright to the authors of this library, please remove this copyright notice. Thank you!